spec/dummy/app/helpers/application_helper.rb

```diff
@@ -8,4 +8,8 @@ def embedded_svg filename, options={}
     end
     doc.to_html.html_safe
   end
+
+  def hide_published_at(post)
```

Comment:
Unused method argument - `post`. If it's necessary, use `_` or `_post` as an argument name to indicate that it won't be used. You can also write as `hide_published_at(*)` if you want the method to accept any arguments but don't care about them.